akanda/rug/test/unit/test_main.py

```diff
@@ -103,6 +104,10 @@ def test_ceilometer_enabled(self, shuffle_notifications, health,
 @mock.patch('akanda.rug.api.quantum.get_local_service_ip')
 class TestMainExtPortBinding(unittest.TestCase):
 
+    @unittest.skipIf(
+        not distutils.spawn.find_executable('ip'),
+        'unsupported platform'
+    )
```

Comment:
This test was failing on OSX, so this should prevent it from running in environments where the `ip` binary is not available.